Acceleratio.XCellKit/ExcelExport/Chart/Chart.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.IO;
+using System.Linq;
+using Acceleratio.XCellKit.ExcelExport.Helpers;
+using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class Charts : ReportExport
+    {
+        public Charts(ReportExport export)
+            : base(export)
+        {
+
+        }
+        /// <summary>
+        /// Creates, designes and places chart into a sheet.
+        /// Calls overrideable methods for displaying the chart based on ChartType.
+        /// </summary>
+        public void InsertChartInSpreadsheet(Stream stream)
```

Comment:
DrawingPart can only be appended to WorksheetPart (constructor is protected internal) 
So I can make AddChart have WorksheetPart parameter or have SpredsheetChart with WorksheetPart property... That way calling AddChart would set chart to worksheetPart.... 

Is that ok?